bugbug/github_dbs_config.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'd like to find another way than having a module listing all GitHub DBs.
What if we remove this module and call `db.register` in the `GitHub` constructor? The path for the DB could be programmatically built like this:
```
https://community-tc.services.mozilla.com/api/index/v1/task/project.bugbug.data_github_{owner}_{repo}_issues.latest/artifacts/public/github_{owner}_{repo}_issues.json.zst
```

Reply:
Sounds good, I'll move it to the same file where GitHub class is, but probably not inside the constructor since `db.register` will be called each time Github instance is created?

Reply:
I could probably check whether it's registered with `db.is_registered()` and not register it again. However there is also db version, which could be different for every DB, so it seems that some sort of config is still needed 🤔

Reply:
The version could be a constant in the `GitHub` module, since the version only changes when we make changes to the way we store data for GitHub repos, so it sill be the same for any GitHub DB.

Reply:
Thanks! I've made it a constant value.